table_processing/benchmark_pipeline.py

```diff
@@ -36,8 +41,8 @@
 summary_df.loc[summary_df.index == 'count'] = summary_df.loc[summary_df.index == 'count'].astype(float).astype(int).astype(str)
 summary_df = summary_df.reset_index().rename(columns={'index':''})
 metrics_df = metrics_df.apply(lambda s: s.apply('{0:.3f}'.format)).reset_index().rename(columns={'index':''})
-#print(metrics_df)
-print(summary_df)
+logging.warning(pd.DataFrame(np.row_stack((summary_df.columns, summary_df.to_numpy())), 
+                columns=['']*len(summary_df.columns), index=['']*(1+len(summary_df))))
```

Comment:
What does this logging message do? Looks like it's a new dataframe? What does this give us?